pkg/api/config/config.go

```diff
@@ -21,8 +21,9 @@ var (
 
 type StorageConfig struct {
 	RootDirectory string
-	GC            bool
 	Dedupe        bool
+	CloudCache    bool
```

Comment:
No...call this LocalCache preferably (or RemoteCache - could be Cloud, NFS, anything else)

Reply:
Renamed to RemoteCache